### ISSUE TITLE
Wrong eol encoding on UNIX ('\r')

### ISSUE THREAD

**Issue by malaterre** [tabulapdf/tabula-java#120]
For some reason tabula is using '\r' to encode an end of line. So one could get something like this in the json encoded output:

`"text" : "The value is same as Diffusion b-value\rof MR Diffusion Macro",
`
However `'\r'` has a funky behavior on UNIX since it will return at the begining of the line. Therefore the equivalent CSV output will be:

`of MR Diffusion Macro,xx30) FD The value is same as Diffusion b-value
`

I would suggest you make sure to use `'\r\n'` to encode an end of line, otherwise the CSV output is pretty much useless on UNIX.

**Comment by malaterre** [comment]
I was asked to reopen the issue initially reported [here](https://github.com/tabulapdf/tabula/issues/570).

**Comment by jeremybmerrill** [comment]
@malaterre Thank you! We'll take a look when we get the chance.

I suspect the issue may be that the extraction algorithm is joining two TextChunks together in the wrong order when it combines them into a Cell.

**Comment by malaterre** [comment]
I do not believe the code is wrong, the issue is really in usage of `\r`. Eg. http://stackoverflow.com/a/14975701/136285

**Comment by malaterre** [comment]
Just a note for anyone reading this, simply add:

```
| tr '\r' ' '
```

after your tabula command
